script/prerender/fleet.ts

```diff
@@ -98,7 +171,7 @@ async function createWorkersStack() {
 
   console.log(`Creating ${workersStackName} stack...`);
 
-  await cfnClient.send(new CreateStackCommand({
+  await sendWithRetries(cfnClient, new CreateStackCommand({
```

Comment:
I tried using `callWithRetries` with `cfnClient.send.bind(cfnClient)` but that runs into some error deep within the AWS libraries, likely due to the fact that `send()` returns different types for different arguments. I could only solve it by making the overloaded `sendWithRetries` using the AWS-provided types.